tests/integration/targets/vmware_guest_instant_clone/tasks/main.yml

```diff
@@ -0,0 +1,385 @@
+# Test code for the vmware_guest_instant_clone Operations.
+# Copyright: (c) 2021, Anant Chopra <chopraan@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- import_role:
```

Comment:
Use block like 

```
https://github.com/ansible-collections/community.vmware/blob/9ebc866169cc1e6ca83db12272dfc02a46cba8a3/tests/integration/targets/vcenter_license/tasks/main.yml#L5
```
This will skip the test case for govcsim.